client/assets/stylesheets/style.scss

```diff
@@ -6,6 +6,8 @@
 
 // Shared
 @import 'shared/reset'; // css reset before the rest of the styles are defined
+
+// Shared
```

Comment:
I think these two lines can be removed now -- there's a `// Shared` heading on L7 already :slightly_smiling_face: